### DISCUSSION TITLE
IPC_MOTION_START_C mq ?

### DISCUSSION THREAD

**Discussion by floronxt** [roleoroleo/yi-hack-Allwinner-v2#947] [General]
Hello roleo,

I am interested in reading the message queue to create a log. I came across `IPC_MOTION_START_C “\x04\x00\x00\x00\x02\x00\x00\x00\x09\x70\x09\x70\x00\x00\x00\x00”`.
Could you please explain in which situation IPC_MOTION_START_C occurs or what the difference is compared to the normal IPC_MOTION_START event?

**Comment by roleoroleo** [comment]
If I remember correctly IPC_MOTION_C is the event used by the cams with fw 12.x
But I'm not completely sure, it's a piece of code written many years ago

**Reply by floronxt** [reply]
Thank you very much for your help. I only have cameras with firmware version 9, which would also explain why I could never find an "IPC_MOTION_START_C" event. Thank you!
